### ISSUE TITLE
org-ml-parse-headlines: require spaces after asterisks

### ISSUE THREAD

**Issue by neeasade** [ndwarshuis/om.el#42]
I'm getting the points of headlines to use with `org-ml-update-headline-at` and found a false positive when using `(org-ml-parse-headlines 'all)`: 

``` org
*this is some bold text* in a paragraph

* this is a headline
```

Both of these get returned as headlines 

I think this is the pattern?: https://github.com/ndwarshuis/org-ml/blob/a2f710129e102dba03004c4bdd84dd576dd98f2e/org-ml.el#L6988

**Comment by ndwarshuis** [comment]
fixed with 466278d09fffc074f582febf8febfb1f2ea73f0e

**Comment by neeasade** [comment]
thanks :sun_with_face:
